glass/src/libnt/native/cpp/NetworkTablesSettings.cpp

```diff
@@ -78,7 +78,7 @@ void NetworkTablesSettings::Thread::Main() {
       if (m_dsClient) {
         wpi::nt::StartDSClient(m_inst, m_port);
       }
-    } else if (m_mode == 3) {
+    } else if (m_mode == 2) {
```

Comment:
probably all of these numbers should be using NT_NET_MODE_*?

Reply:
They correspond with the indices of the values in the enum chooser array [here](https://github.com/wpilibsuite/allwpilib/blob/632eb0922de23a7db1a99577f2fa57904da03b0e/glass/src/libnt/native/cpp/NetworkTablesSettings.cpp#L90), so I don't think using the NT mode enum is the right choice. Totally agree it'd be better if these weren't magic numbers, but I'm not sure how to link the selection result back to the input choices with the current api shape. 
Perhaps EnumSetting should take an array of pairs of (display string, value)?